NativeYoutube.xcodeproj/project.pbxproj

```diff
@@ -584,12 +588,12 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		C22D70B629557E71000DE71E /* XCRemoteSwiftPackageReference "SDWebImageSwiftUI" */ = {
+		70A3843D2955D1A1000941F5 /* XCRemoteSwiftPackageReference "YouTubeKit" */ = {
```

Comment:
Are you working on a stale (old) branch?

Reply:
You mean for YouTubeKit ?

Reply:
Trying Pulling the latest codebase :)

Reply:
If you are speaking about the app's repo, i'm sure i work on the right branch and for YouTubeKit it's simply because i can't find another code snippet/module to extract streams from yt links :0